analytics/sam/app/template.yaml

```diff
@@ -29,7 +29,7 @@ Resources:
     Properties:
       Location:
         ApplicationId: arn:aws:serverlessrepo:us-east-1:646794253159:applications/aws-dynamodb-stream-eventbridge-fanout
-        SemanticVersion: 1.0.0
+        SemanticVersion: 1.0.1
```

Comment:
We should merge [the PR](https://github.com/awslabs/aws-dynamodb-stream-eventbridge-fanout/pull/15) in this repo and upgrade it to 1.0.2 before updating this.

Reply:
I went ahead and merged that PR and updated the app to 1.0.2 so you can change this to reference 1.0.2 now.